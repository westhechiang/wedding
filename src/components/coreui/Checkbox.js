import styled from 'styled-components';
import { theme } from '../../ui';

const Checkbox = styled.input.attrs({
  type: 'checkbox',
})`
  color: ${theme.color.white};
  background: transparent;
  margin: 0 5px 0 0;
`;

export default Checkbox;
