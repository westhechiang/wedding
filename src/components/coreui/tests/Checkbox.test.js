import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Checkbox from '../../../components/coreui/Checkbox';

describe('<Checkbox />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<Checkbox />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
