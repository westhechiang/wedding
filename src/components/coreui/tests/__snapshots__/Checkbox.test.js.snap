// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> should match its empty snapshot 1`] = `
<input
  className="sc-bdVaJa cRadJj"
  type="checkbox"
/>
`;
